test/unit/modules/blocks/api.js

```diff
@@ -14,87 +14,81 @@
 
 'use strict';
 
-var rewire = require('rewire');
-var modulesLoader = require('../../../common/modules_loader');
-var application = require('../../../common/application'); // eslint-disable-line no-unused-vars
+const rewire = require('rewire');
 
-var BlocksApi = rewire('../../../../modules/blocks/api.js');
+const BlocksApi = rewire('../../../../modules/blocks/api.js');
 
 describe('blocks/api', () => {
-	var blocksApi;
-	var __private;
-	var dbStub;
-	var blocksApiModule;
-
-	before(done => {
-		application.init(
-			{ sandbox: { name: 'lisk_test_blocks_api' }, waitForGenesisBlock: true },
-			(err, scope) => {
-				blocksApi = scope.modules.blocks.shared;
-				blocksApi.onBind(scope.modules);
-
-				done();
-			}
+	let __private;
+	let blocksApiModule;
+	let library;
+	let loggerStub;
+	let dbStub;
+	let blockStub;
+	let schemaStub;
+	let dbSequenceStub;
+
+	beforeEach(done => {
+		dbStub = {
+			blocks: {
+				list: sinonSandbox.stub().resolves([]),
+				sortFields: [
+					'id',
+					'timestamp',
+					'height',
+					'previousBlock',
+					'totalAmount',
+					'totalFee',
+					'reward',
+					'numberOfTransactions',
+					'generatorPublicKey',
+				],
+			},
+		};
+
+		loggerStub = {
```

Comment:
can you rename this to `loggerSpy` as the code suggest we are just spying on the log methods.

Reply:
I will do it.